io.github.mahmoudbahaa.outlook_for_linux.yml

```diff
@@ -0,0 +1,57 @@
+app-id: io.github.mahmoudbahaa.outlook_for_linux
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: outlook-for-linux
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
```

Comment:
Why do you need full home? Does it support filechooser portals?

Reply:
removed